README.md

```diff
@@ -1,24 +1,89 @@
-# README
+# Budget App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> A mobile web application where a user manage its budget: have a list of transactions associated with a category, so that the user can see how much money it spent and on what.
 
-Things you may want to cover:
+## Live Link
 
-* Ruby version
+- [Heroku](https://lit-ridge-01801.herokuapp.com/)
 
-* System dependencies
+## Video
 
-* Configuration
+- [Video](https://www.loom.com/share/58e07b401609459dbded6b3987db7380)
 
-* Database creation
+## Built With
 
-* Database initialization
+- Ruby
+- Ruby on Rails
+- Postgresql
+- Bootstrap
 
-* How to run the test suite
+## Entity Relationship Diagram
 
-* Services (job queues, cache servers, search engines, etc.)
+![screenshot](./app/assets/images/erd_diagram.png)
 
-* Deployment instructions
+## Getting Started
+
+**To get a local copy up and running follow these simple example steps:**
+### Prerequisites
+
+- [Ruby](https://www.ruby-lang.org/en/)
+- [Rails](https://gorails.com/)
+
+### Setup
+
+- Make sure you have Ruby on Rails set up properly on your computer
+
+``` sh 
+  git git@github.com:ridaarif98/Budget-app.git
+``` 
+``` sh 
+  cd Budget-app
+```
+
+### Install
+
+```sh
+  bundle install
+```
+
```

Comment:
- [ ] Please include one more command which is `npm install`, otherwise I'm getting issues in running the application.